File: go/iq/fbid-devices~get~user+users.go
// BuildDevicesFetch builds <iq type="get" xmlns="fbid:devices">. Replies with one of: Success, Error.
func BuildDevicesFetch() waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": "s.whatsapp.net", "xmlns": "fbid:devices", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "users", Attrs: waBinary.Attrs{}},
		},
	}
}

File: rust/iq/fbid-devices~get~user+users.rs
/// <iq type="get" xmlns="fbid:devices">. Replies with one of: Success, Error.
pub fn build_devices_fetch() -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "fbid:devices")
        .attr("type", "get")
        .child(NodeBuilder::new("users").build())
        .build()
}